deploy-board/deploy_board/webapp/templatetags/utils.py

```diff
@@ -138,6 +138,16 @@
     "STOPPED": "Completed stopping",
 }
 
+_AGENT_STATES_TO_TIPS = {
+    "RESET": "Agent is retrying",
```

Comment:
User requested to restart the service

Reply:
leaving the naming the same for now and will iterate to what you've suggested after user feedback